package/Dockerfile

```diff
@@ -1,19 +1,40 @@
 ARG K3S_BUILDER=rancher/k3s:v1.25.9-k3s1
 FROM ${K3S_BUILDER} as k3s_builder
 
-FROM ghcr.io/oracle/oraclelinux:7-slim
-
-RUN yum-config-manager --enable ol7_optional_latest && \
-    yum-config-manager --enable ol7_addons && \
-    yum update -y && \
-    # software collections repo needed for git 2.x on OL7
-    yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
-    yum install -y rh-git227 curl ca-certificates unzip tar docker-cli nc mkisofs && \
-    yum clean all && \
-    rm -rf /var/cache/yum && \
-    # Set up needed to ensure git 2.27 from rh-git227 is on the path
-    ln /opt/rh/rh-git227/enable /etc/profile.d/git.sh && \
-    source /etc/profile.d/git.sh && \
+FROM ghcr.io/oracle/oraclelinux:8-slim as build
+ARG ARCH
+ENV ARCH=$ARCH
+ENV HELM_VERSION 3.11.3-rancher1
+
+COPY . /go/src/github.com/rancher/rancher/
+
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
+    microdnf install -y git make findutils && \
+    microdnf install -y golang-1.20.10-1.module+el8.9.0+90068+08775a1f.x86_64 && \
+    cd /go/src/github.com/rancher/rancher/scripts && \
+    ./build && \
+    ./k3s-images.sh && \
+    # Rancher uses its own Helm fork - build and install our BFS of that fork.
+    git clone --branch oracle/release/${HELM_VERSION} --depth=1 https://github.com/verrazzano/helm && \
+    make -C ./helm
+
+FROM ghcr.io/oracle/oraclelinux:8-slim
+
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
+    microdnf install -y git gcc vim file wget unzip jq nc mkisofs && \
+    microdnf install -y glibc-static --enablerepo=ol8_codeready_builder && \
+    microdnf install -y openssl-devel libffi-devel python3 iproute bash-completion sysstat hostname && \
+    microdnf install -y dnf && \
```

Comment:
since you are installing dnf , install everything else with dnf by moving this up and reducing the number of "microdnf install -y" statements with a single "dnf install -y" statement